portal-kernel/src/com/liferay/portal/kernel/portlet/BaseJSPSettingsConfigurationAction.java

```diff
@@ -90,15 +90,13 @@ public void include(
 		}
 	}
 
-	public void setServletContext(ServletContext servletContext) {
-		_servletContext = servletContext;
-	}
-
 	protected ServletContext getServletContext(
 		HttpServletRequest httpServletRequest) {
 
-		if (_servletContext != null) {
-			return _servletContext;
+		ServletContext servletContext = getServletContext();
```

Comment:
@W2jx This one is a bit special. The `getServletContext` method comes from `LiferayPortlet` and it has default logic. Could you double check whether the default logic can return the same servlet context you referenced?

Reply:
Ok, I will double check it.